### ISSUE TITLE
SVT:Failed to start server in debug mode and timeouts are not configurable 

### ISSUE THREAD

**Issue by jagraj** [OpenLiberty/liberty-tools-eclipse#283]
I started my application in debug mode and I got into following error. I think my application takes time to start and there is no option for me to configure timeouts to start or debug application.

```
Unable to attach debugger to host: localhost and port: 50434
Unable to connet to JVM on host: localhost and port: 50434
```

![image](https://media.github.ibm.com/user/26419/files/7d7b6f67-90d9-4214-abb4-3880165e79ae)

**Comment by rumanaHaque** [comment]
I do see the same issue with the ACME application as well.
When I try to debug my app using "Debug As" (As shown below)

![image](https://user-images.githubusercontent.com/31224827/206301620-8c9990ec-8378-4d21-948d-21113bcfcb8b.png)

My application starts, but it fails to attach to the Debugger.

![image](https://user-images.githubusercontent.com/31224827/206301828-34bee858-7307-4179-a499-8c3bc168c7fa.png)

**Comment by jagraj** [comment]
After investigating this problem with the development team, we found out that liberty tools does not support liberty maven plugin `<installDirectory>` feature. I switched `<installDirectory>` option to `<runtimeArtifact>` then the debug feature started working. We need to display valid error message to the user when we fail to attach debugger.

I opened this new issue(https://github.com/OpenLiberty/liberty-tools-eclipse/issues/291) and requested debug support for existing liberty installs in my developer machine.
